libraries/AP_ServoRelayEvents/AP_ServoRelayEvents.cpp

```diff
@@ -36,8 +36,11 @@ bool AP_ServoRelayEvents::do_set_servo(uint8_t _channel, uint16_t pwm)
         // cancel previous repeat
         repeat = 0;
     }
-    hal.rcout->enable_ch(_channel-1);
-    hal.rcout->write(_channel-1, pwm);
+    SRV_Channel *c = SRV_Channels::srv_channel(_channel-1);
+    if (c == nullptr) {
```

Comment:
that's right. It is actually deliberate, and matches previous behaviour. For example, you can DO_SET_SERVO on a bottle drop function channel.